modules/codemod/lib/softDeprecate/Stack/spec/softDeprecateStack.spec.ts

```diff
@@ -0,0 +1,312 @@
+import {expectTransformFactory} from './expectTransformFactory';
+import transformer from '../softDeprecateStack';
+const context = describe;
+
+const expectTransform = expectTransformFactory(transformer);
+
+describe('Canvas Kit Deprecate Stack Codemod', () => {
+  context('when transforming Stack imports', () => {
+    it('should ignore non-canvas-kit imports', () => {
+      const input = `import {Stack} from '@workday/some-other-library'`;
+      const expected = `import {Stack} from '@workday/some-other-library'`;
+
+      expectTransform(input, expected);
+    });
+
+    it('should properly transform named imports from @workday/canvas-kit-react', () => {
+      const input = `
+        import {
+          Stack,
+          HStack,
+          VStack,
+          StackProps,
+          HStackProps,
+          VStackProps,
+          StackStyleProps,
+        } from '@workday/canvas-kit-react'
+      `;
+      const expected = `
+        import {
+          Flex,
+          Flex,
+          Flex,
+          FlexProps,
+          FlexProps,
+          FlexProps,
+          FlexProps,
+        } from '@workday/canvas-kit-react'
+      `;
+
+      expectTransform(input, expected);
+    });
+
+    it('should properly transform named imports from @workday/canvas-kit-react/layout', () => {
+      const input = `
+        import {
+          Stack,
+          HStack,
+          VStack,
+          StackProps,
+          HStackProps,
+          VStackProps,
+          StackStyleProps,
+        } from '@workday/canvas-kit-react/layout'
+      `;
+      const expected = `
+        import {
+          Flex,
+          Flex,
+          Flex,
+          FlexProps,
+          FlexProps,
+          FlexProps,
+          FlexProps,
+        } from '@workday/canvas-kit-react/layout'
```

Comment:
I think the transform should just import this once

Reply:
Definitely. This is what I'm working on now

Reply:
This is fixed